internal/fixperms/fdfs/fdfs.go

```diff
@@ -102,6 +112,16 @@ func (s *FS) RecursiveChown(uid, gid int) error {
 	}
 	for _, d := range ds {
 		if !d.IsDir() {
+			// Skip lchown if the uid and gid already match. This avoids updating
+			// the ctime of files unnecessarily.
+			var stat *unix.Stat_t
+			if stat, err = s.Stat(d.Name()); err != nil {
```

Comment:
Being inside a new scope, we can freely redeclare `err` (which has the extra upside of hinting that it's unrelated to the previous `err`). So I'd write this:

```suggestion
			stat, err := s.Stat(d.Name())
			if err != nil {
```